leader-election-impl/src/main/java/com/palantir/leader/PaxosLeaderElectionService.java

```diff
@@ -345,7 +345,10 @@ private void proposeLeadership(LeadershipToken token) {
             proposer.propose(seq, null);
         } catch (PaxosRoundFailureException e) {
             // We have failed trying to become the leader.
-            leaderLog.warn("Leadership was not gained", e);
+            leaderLog.warn("Leadership was not gained because: " + e.getMessage() + ". "
```

Comment:
Isn't the goal of this change to avoid log spam and not log the full stacktrace when leadership is not gained? If so, the current version of this change seems fine to me.